core/trino-main/src/test/java/io/trino/block/BlockAssertions.java

```diff
@@ -660,18 +681,12 @@ public static Block createDoublesBlock(Double... values)
 
     public static Block createDoublesBlock(Iterable<Double> values)
     {
-        BlockBuilder builder = DOUBLE.createBlockBuilder(null, 100);
-
-        for (Double value : values) {
-            if (value == null) {
-                builder.appendNull();
-            }
-            else {
-                DOUBLE.writeDouble(builder, value);
-            }
-        }
+        return createDoublesBlock(DOUBLE, values);
+    }
 
-        return builder.build();
+    public static Block createDoublesBlock(Type type, Iterable<Double> values)
```

Comment:
type is not needed here, is it? Only `DOUBLE` uses `double` primitive

Reply:
yup, it's not needed. I also remove unnecessary type from other methods